test/integration/world_location_integration_test.rb

```diff
@@ -37,6 +37,51 @@ def put_content_hash_containing(locale, title, mission_statement)
     has_entries(locale: locale, title: title, details: has_entries(mission_statement: mission_statement))
   end
 
+  test "when updating, makes the correct calls to search api and publishing api" do
+    Sidekiq::Testing.inline! do
+      world_location_without_translations = create(:world_location, news_page_content_id: "id-123")
+      visit edit_admin_world_location_path(world_location_without_translations)
+      new_mission_statement = "a different mission"
+      fill_in "world_location_mission_statement", with: new_mission_statement
+
+      presenter = PublishingApi::WorldLocationNewsPresenter.new(world_location_without_translations)
+
+      Services.publishing_api.expects(:put_content).once.with("id-123", put_content_hash_containing("en",  world_location_without_translations.title, new_mission_statement))
+      Services.publishing_api.expects(:put_content).once.with(world_location_without_translations.content_id, has_entries(document_type: "world_location"))
+      Whitehall::FakeRummageableIndex.any_instance.expects(:add).with(presenter.content_for_rummager("id-123"))
+      Services.publishing_api.expects(:publish).at_least_once
+
+      click_on "Save"
+    end
+  end
+
+  test "when updating an inactive location, does not make any calls to search api" do
```

Comment:
totally not the place for this chat, but what's an inactive location?

Reply:
From what I can see, we've imported the list of World Locations from somewhere.  For some of these (e.g. the UK), we don't want a world location page, so they are marked as 'inactive'.  Although we weirdly still have a location page published, presumably for legacy reasons 😕.